<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dove-eye</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>782</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="gui::FramesetViewer" name="viewer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="widgets::SceneViewer" name="scene_viewer" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="widgets::PlaybackControl" name="playback_control" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_parameters">
    <property name="title">
     <string>&amp;Parameters</string>
    </property>
    <addaction name="action_parameters_modify"/>
    <addaction name="separator"/>
    <addaction name="action_parameters_load"/>
    <addaction name="action_parameters_save"/>
   </widget>
   <widget class="QMenu" name="menu_providers">
    <property name="title">
     <string>P&amp;roviders</string>
    </property>
    <addaction name="action_setup_cameras"/>
    <addaction name="action_open_video_files"/>
   </widget>
   <widget class="QMenu" name="menu_localization">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Localization</string>
    </property>
    <addaction name="action_localization_start"/>
    <addaction name="action_localization_stop"/>
    <addaction name="action_localization_save"/>
   </widget>
   <widget class="QMenu" name="menu_calibration">
    <property name="title">
     <string>&amp;Calibration</string>
    </property>
    <widget class="QMenu" name="menu_distortion">
     <property name="title">
      <string>Distortion</string>
     </property>
     <addaction name="action_distortion_ignore"/>
     <addaction name="action_distortion_video"/>
     <addaction name="action_distortion_data"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="action_calibrate"/>
    <addaction name="action_abort_calibration"/>
    <addaction name="separator"/>
    <addaction name="action_calibration_load"/>
    <addaction name="action_calibration_save"/>
    <addaction name="separator"/>
    <addaction name="menu_distortion"/>
   </widget>
   <widget class="QMenu" name="menu_scene_viewer">
    <property name="title">
     <string>Scene viewer</string>
    </property>
    <addaction name="action_scene_show_cameras"/>
    <addaction name="action_scene_clear_trajectory"/>
   </widget>
   <widget class="QMenu" name="menu_tracker">
    <property name="title">
     <string>Tracker</string>
    </property>
   </widget>
   <addaction name="menu_providers"/>
   <addaction name="menu_calibration"/>
   <addaction name="menu_localization"/>
   <addaction name="menu_parameters"/>
   <addaction name="menu_scene_viewer"/>
   <addaction name="menu_tracker"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_parameters_modify">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="action_setup_cameras">
   <property name="text">
    <string>&amp;Cameras</string>
   </property>
  </action>
  <action name="action_calibration_load">
   <property name="text">
    <string>Load calibration data</string>
   </property>
  </action>
  <action name="action_calibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_calibration_save">
   <property name="text">
    <string>Save calibration data</string>
   </property>
  </action>
  <action name="action_abort_calibration">
   <property name="text">
    <string>Abort calibration</string>
   </property>
  </action>
  <action name="action_localization_start">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="action_localization_stop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action_localization_save">
   <property name="text">
    <string>Save localization data</string>
   </property>
  </action>
  <action name="action_open_video_files">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video &amp;file(s)</string>
   </property>
  </action>
  <action name="action_parameters_load">
   <property name="text">
    <string>Load parameters</string>
   </property>
  </action>
  <action name="action_parameters_save">
   <property name="text">
    <string>Save parameters</string>
   </property>
  </action>
  <action name="action_distortion_ignore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Ignore</string>
   </property>
  </action>
  <action name="action_distortion_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Undistort &amp;video</string>
   </property>
  </action>
  <action name="action_distortion_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Undistort &amp;data only</string>
   </property>
  </action>
  <action name="action_scene_show_cameras">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show cameras</string>
   </property>
  </action>
  <action name="action_scene_clear_trajectory">
   <property name="text">
    <string>Clear trajectory</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::FramesetViewer</class>
   <extends>QWidget</extends>
   <header>gui/frameset_viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>widgets::SceneViewer</class>
   <extends>QWidget</extends>
   <header>widgets/scene_viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>widgets::PlaybackControl</class>
   <extends>QWidget</extends>
   <header>widgets/playback_control.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
